client/container.go

```diff
@@ -24,15 +24,6 @@ type PrmContainerPut struct {
 
 	sessionSet bool
 	session    session.Container
-
-	signer neofscrypto.Signer
-}
-
-// SetSigner sets signer to sign request payload.
-// Signer's scheme MUST be neofscrypto.ECDSA_DETERMINISTIC_SHA256. For example, you can use neofsecdsa.SignerRFC6979.
```

Comment:
we must not loose this requirement in method docs

Reply:
@smallhive good, but one note: this restriction applies only to container operations now (as an additional benefit, this allows us to exec object ops with different schemes)

Reply:
Got it, removed from object operations